### PR TITLE
Add GitHub Issue Forms

### PR DESCRIPTION
**Pull request by wdawson**
Improves UX when submitting a bug. Happy to change anything in the template as needed.

issue form

Closes INS-907

### CODE REVIEW

**Review by reynolek — APPROVED** [review]
beautiful

**Review by develohpanda — APPROVED** [review]
Looks awesome, thank you for doing this!

**Comment by develohpanda** [comment]
Hey @wdawson, is this good to go, or are you waiting on reviews? Feel free to merge when ready 🙌🏽

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+      - label: I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for a bug that matches the one I want to file, without success.
+        required: true
+- type: input
+  attributes:
+    label: Insomnia Version
+    description: What version of Insmonia are you using?
+    placeholder: "2021.5"
```
Comment (dimitropoulos) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    placeholder: "2021.5.0"
```

There is no version of insomnia that matches this - it seems best to not avoid confusion and imply that people use actual version numbers.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    label: Installation method
+    description: How did you install Insomnia?
+    placeholder: "e.g. download from insomnia.rest, homebrew, apt, etc."
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Last Known Working Insomnia version
+    description: What is the last version of Insomnia this worked in, if applicable?
+    placeholder: "2021.4"
```
Comment (dimitropoulos) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    placeholder: "2021.4.0"
```

Same suggestion as above regarding valid versions.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+      - macOS
+      - Ubuntu
+      - Other Linux
+      - Other (specify below)
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Operating System Version
+    description: What operating system version are you using? On Windows, click Start button > Settings > System > About. On macOS, click the Apple Menu > About This Mac. On Linux, use lsb_release or uname -a.
```
Comment (dimitropoulos) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
    description: |-
      What operating system version are you using?
      On Windows, click Start button > Settings > System > About.
      On macOS, click the Apple Menu > About This Mac.
      On Linux, use `lsb_release` or `uname -a`.
```

image

**Pull request merged**